spec/helpers/application_helper_spec.rb

```diff
@@ -411,4 +411,22 @@
     end
   end
 
+  describe 'icon_link' do
+    it 'returns the HTML for a link to the url with the FontAwesome (fa) icon name given, target = blank (open in a new window)' do
+      expect(helper.icon_link('http://example.com', 'facebook-square')).to eq "<a target=\"_blank\" href=\"http://example.com\"><i class='fab fa-facebook-square fa-2x'></i></a>"
```

Comment:
The `icon()` method in the FontAwesome Sass gem will return a string with double quotes, so we should expect that:
```
 expect(helper.icon_link('http://example.com', 'facebook-square')).to eq "<a target=\"_blank\" href=\"http://example.com\"><i class=\"fab fa-facebook-square fa-2x\"></i></a>"
```